superset/assets/src/visualizations/core/registries/ChartComponentLoaderRegistry.js

```diff
@@ -0,0 +1,10 @@
+import LoaderRegistry from '../../../modules/LoaderRegistry';
+import makeSingleton from '../../../utils/makeSingleton';
+
+class ChartComponentLoaderRegistry extends LoaderRegistry {
+  constructor() {
+    super('ChartComponent');
+  }
+}
+
+export const getInstance = makeSingleton(ChartComponentLoaderRegistry);
```

Comment:
why not prefer `default` exports for all of the singletons?

Reply:
This one I struggle with a bit. Usually when it's a class. I expect the `default` to be the `class`, but since this is the `getter` for the singleton. It may be misleading. 

What is the common practice around here for singleton API? I am open to alternatives, as long as we keep this consistent.

Reply:
Will rename the file to `XXXSingleton`